infra/recipes/k8s/environments/azure/sandbox/housetables/values.yaml

```diff
@@ -0,0 +1,34 @@
+namespace: "default"
+
+openhouseClusterTag: "openhouse"
+
+clusterPropertiesPath: "config/azure/sandbox/cluster.yaml"
+
+htsService:
+  replicaCount: 1
+  image:
+    repository: "containerregistryopenhouse.azurecr.io/openhouse-housetables-service"
+    tag: "latest"
+    pullPolicy: IfNotPresent
+  service:
+    type: ClusterIP
+    port: 8080
+  port: 8080
+  resources:
+    requests:
+      cpu: 1
+      memory: 1G
+    limits:
+      cpu: 1
+      memory: 2G
+  mysql:
+    enabled: true
+    secrets:
+      HTS_DB_USER: MYSQL_USER
+      HTS_DB_PASSWORD: MYSQL_PASSWORD
+  command: "java -Xmx1024M -Xms128M -XX:NativeMemoryTracking=summary -cp housetables.jar -Dloader.main=com.linkedin.openhouse.HouseTablesSpringApplication org.springframework.boot.loader.PropertiesLauncher $APP_NAME.jar $@"
+  appName: housetables-service
+
+housetables:
+  database:
+    url: ""
```

Comment:
Is the AKS DB url dynamic? Why not populated here?

Reply:
I removed these lines and then populated from the helm release terraform